doc/classes/CodeEdit.xml

```diff
@@ -168,6 +168,12 @@
 				Duplicates all selected text and duplicates all lines with a caret on them.
 			</description>
 		</method>
+		<method name="fold_all_comments">
+			<return type="void" />
+			<description>
+				Folds all comments that are possible to be folded (see [method can_fold_line]).
```

Comment:
```suggestion
				Folds all comments that can be folded (see [method can_fold_line]).
```

Reply:
Sounds good, it's not a valid sentence so good to fix rather than better wording